src/utils/cache.js

```diff
@@ -0,0 +1,32 @@
+const logger = require('../logger');
+
+class Cache {
```

Comment:
basically this could extend a Set, right?

Reply:
Set as input, set as default, or Cache extend Set?
I hope we can update it in future to a solution that is used redis.